README.md

```diff
@@ -13,20 +13,21 @@ Currently requires
 
 ## Quick Start Instructions
 
-As root: `start.py [-h] [-c] [--max-args MAX_ARGS] [-j] [-e EVENTS_TO_TRACE]`
+As root: `tracee.py [-h] [-c] [--max-args MAX_ARGS] [-j] [-e EVENTS_TO_TRACE]`
 
 optional arguments:
-  -h, --help            show this help message and exit
-  -c, --container       only trace newly created containers
-  --max-args MAX_ARGS   maximum number of arguments parsed and displayed, defaults to 20
-  -j, --json            save events in json format
-  -e EVENTS_TO_TRACE, --events-to-trace EVENTS_TO_TRACE
-                        trace only the specified events and syscalls (default: trace all)
-
-examples:
-    ./start.py -c
-
-Following is an output example of Tracee after running
+&nbsp;&nbsp;-h, --help            show this help message and exit<br>
+&nbsp;&nbsp;-c, --container       only trace newly created containers<br>
+&nbsp;&nbsp;--max-args MAX_ARGS   maximum number of arguments parsed and displayed, defaults to 20<br>
+&nbsp;&nbsp;-j, --json            save events in json format<br>
+&nbsp;&nbsp;-l, --list            list events<br>
+&nbsp;&nbsp;-e EVENTS_TO_TRACE, --events-to-trace EVENTS_TO_TRACE<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;trace only the specified events and syscalls (default: trace all)
+
+examples:<br>
+`./tracee.py -c`
+
+Following is an output example of Tracee after running<br>
```

Comment:
The html code <br> does get interpreted as a break in markdown, but I don't think it's necessary (and would be more consistent not to use it since we don't use it anywhere else)

Reply:
Without it it's harder to read, as it is kept in one line. The only other way for newline that I know about is using 2 spaces in the end of the sentence, but this is also not a good way. as editors may remove trailing spaces

Reply:
You can just put in an extra line break (it's very unusual to see HTML in markdown)